### ISSUE TITLE
FR: Unify the appearance of information windows

### ISSUE THREAD

**Issue by dtcyganok** [koreader/koreader#12838]
**The problem**
There are several very similar windows: book information, book statistic, battery statistics and system statistics, but they look completely different.

**The solution**
Unify the appearance of the aforementioned windows.

**Comment by Frenzie** [comment]
Indeed, it looks like I neglected to add colons to system statistics back in 2017. Completely different is a bit dramatic though. ;-)

**Comment by poire-z** [comment]
> The biggest difference is that some of them reduce the font size to fit the content on one screen. 

I think I explained it in https://github.com/koreader/koreader/issues/12817#issuecomment-2509385995.

> P.S. Probably I should've created this issue as a bug.

Don't be so sure everything you don't like is a bug. We did put some thinking in most things.

**Comment by mergen3107** [comment]
> Probably I should've created this issue as a bug.

I think FR is better, because this isn't a bug where things break or work unexpectedly. It is just the design is different

**Comment by Frenzie** [comment]
Battery statistics and anything else single-page on landscape is probably a bug.

> Don't be so sure everything you don't like is a bug. We did put some thinking in most things.

Or put another way, nothing is simper than keeping all font sizes identical. Making them different takes effort, which implies certain things. In this case I'm inclined to agree with whoever put in the effort that having some 1-3 items on page 2 is kind of annoying, especially on older eink devices.

**Comment by dtcyganok** [comment]
> Don't be so sure everything you don't like is a bug. We did put some thinking in most things.

I really don't want it to look like opinionated nitpicking... I know you put a lot of thinking in it. Actually I highly appreciate koreader, and want to help make it better.

Except different appearance, in my opinion there are also some design problems with single_page in landscape mode:
1. Font size became too small, and it hurts legibility
2. The distance between left and right columns is big, thus it's difficult to track the line

It might be worth considering a two-column layout on landscape

**Comment by Frenzie** [comment]
Much easier said than done. ^_^ But avoiding single-page in landscape is presumably simple enough.
